stagecraft/apps/dashboards/models/dashboard.py

```diff
@@ -250,9 +250,14 @@ def spotlightify(self, request_slug=None):
         return modules_or_tabs
 
     def serialize(self):
+        def simple_field(field):
+            return not (field.is_relation or field.one_to_one or (
```

Comment:
This seems different to https://docs.djangoproject.com/en/1.8/ref/models/meta/#migrating-old-meta-api

- MyModel._meta.get_fields_with_model():

```python
[
    (f, f.model if f.model != MyModel else None)
    for f in MyModel._meta.get_fields()
    if not f.is_relation
        or f.one_to_one
        or (f.many_to_one and f.related_model)
]
```

Should it be:

```python
return not field.is_relation or field.one_to_one or (field.many_to_one and field.related_model))
```